rest/snippets/UserRecordings/RecordingEntity.yml

```diff
@@ -97,6 +97,8 @@ RecordingEntity:
           properties:
             data:
               type: object
+              nullable: true
+              description: May be null if recording was not created as an automated recording.
```

Comment:
This relationship doesn't have anything to do with automated recording, If the recording was created individually by the user it could still have this relationship. It's nullable only if the recorded program does not belong to any series.